src/Main.java

```diff
@@ -4,6 +4,9 @@ public static void main(String[] args) {
         Animal a1 = new Dog();
         a1.makeSound();
 
+        Animal a2 = new Sheep();
```

Comment:
please try to use a descriptive name for the variable instead of a2.

Reply:
solved, thanks for the feedback